x/oracle/types/events.go

```diff
@@ -3,6 +3,8 @@ package types
 const (
 	EventTypeRegistration              = "oracle_registration"
 	EventTypeApproveOracleRegistration = "approve_oracle_registration"
+	EventTypeUpgrade                   = "oracle_upgrade"
+	EventTypeApproveOracleUpgrade      = "approve_oracle_upgrade"
```

Comment:
Also, add separate event for oracle upgrade. This event type will be used in oracle side.